task_14/js/task4.js

```diff
@@ -0,0 +1,23 @@
+let a = prompt('введите первое число');
+let e = prompt('введите операцию');
+let b = prompt('введите второе число');
+let result;
+a = +a;
+b = +b;
+if (e == '+'){
```

Comment:
You can use if without else and your code will more readable
`
if (e == '-') {
   ...
}
if (e == '*') {
   ...
}
`

Reply:
done